### ISSUE TITLE
[Suggestion]:  Adding possibility for custom paintings to custom dimensions, through datapack or config? We could make the paintings ourselves or something

### ISSUE THREAD

**Issue by WelpSTudent** [Mrbysco/Dimensional-Paintings#8]
### General Info

- [X] I would like to propose new content
- [ ] I would like to propose changes
- [ ] I would like to propose another thing

### Suggestion Description

For dimensions from mods like Beyond Earth

**Comment by digital-pet** [comment]
I'm also interested in this, specifically compatibility with the dimension added by the lost cities mod.

**Comment by Winddbourne** [comment]
Exactly, mods like the lost cities need this.  It supports the dimension but provides no way to get there and back.  I also would like it for all of those "deep dark" mods that want the portal to be right in the middle of the wardens city, or as a short cut to the end. 

But mostly I need something like this for modded dimensions that either don't have proper portals, or that provide ones that are buggy, inconvenient, or otherwise annoying.
